src/style.css

```diff
@@ -0,0 +1,18 @@
+body {
+  margin: 0;
+}
+
+#red-block {
+  height: 100px;
+  background-color: red;
+}
+
+#green-block {
+  height: 100px;
+  background-color: green;
+}
+
+#blue-block {
+  height: 100px;
+  background-color: blue;
+}
```

Comment:
The name of the element class should make a sense. It should describe what the role of the element is on this page or describe its state. In the case with the name _red_, _green_, maybe in the future, the color of your block will be changed, so you have to rename all classes on the page.

Reply:
The theme of the site may be changed and binding class names to colors is bad practice. You can think of these blocks as content, wrapper, and inner wrapper. Example: outer, inner, wrapper, content, etc.

Reply:
If we have 5 blocks and these blocks have common characteristics, such as height, width, color, fonts, etc., then we should move these characteristics in a separate class and use it for style these blocks. DRY - Don't Repeat Yourself.

Reply:
IDs are used for JS, try not to style the id if you can avoid it. This makes impossible to reuse the code. Stylize classes. Style guide